dsgrid/config/dimensions.py

```diff
@@ -339,7 +339,22 @@ class TimeDimensionModel(TimeDimensionBaseModel):
     )
     ranges: List[TimeRangeModel] = Field(
         title="time_ranges",
-        description="Defines the continuous ranges of time in the data.",
+        description="Defines the continuous ranges of time common for all sets of data series in the dataset.",
+    )
+    ranges_timezone: TimeZone = Field(
+        title="timezone_for_time_ranges",
+        description="""
+        Time zone of the ranges specified:
+            UTC, 
+            HawaiiAleutianStandard, 
+            AlaskaStandard, AlaskaPrevailing,
+            PacificStandard, PacificPrevailing, 
+            MountainStandard, MountainPrevailing, 
+            CentralStandard, CentralPrevailing, 
+            EasternStandard, EasternPrevailing,
+            LOCAL
```

Comment:
This should probably go into an enumeration instead of being duplicated here in the description.

Reply:
Looks like you don't need to add any of these. The docs take care of it:
![image](https://user-images.githubusercontent.com/7425288/147784485-c9c7a342-624c-4b97-8cda-eba9d516f333.png)